addons/website/tests/test_page.py

```diff
@@ -344,7 +344,8 @@ def test_website_homepage_url_change(self):
             'key': 'test.homepage_url_test',
             'url': '/homepage_url_test',
             'is_published': True,
-        })
+        }).with_context(website_id=website.id)
+
```

Comment:
Hello @bso-odoo , Thank you for the review. Setting the website_id during creation wouldn't resolve the issue because the get_current_website function retrieves the website_id from the session or context (with a fallback if neither is available)

Reply:
But then the created page might have been created for the wrong website.

To be clear, what I meant is:
```py
        test_page = self.env['website.page'].with_context(website_id=website.id).create({
```